.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,23 @@
-_Motivation:_  Explain here what the reason for the pull request is.
-_Linked issues:_  Optionally, add links to existing issues or other PR's
+## Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.
+
+Fixes # Optionally, add links to existing issues or other PR's
+
+## Checklist
 
-### Checklist
-- [ ] Build rule `all-supported` completed successfully
 - [ ] Package upgrade completed successfully
 - [ ] New installation of package completed successfully
+- [ ] Package functionality was tested
+- [ ] Any needed documentation is updated
+
+
+### Type of change
+
+Please use any relavent tags.
+
+- [ ] Bug fix
+- [ ] New Package
+- [ ] Package update
+- [ ] Includes framework changes
```

Comment:
When ever possible I would create a different PR template for framework changes.
- we should create dedicated PRs for framework changes (/mk/*) as those do not trigger package builds and require extended build tests
- the checklist for framework changes would look different...
-

Reply:
[GitHub](https://github.community/t/is-there-a-pull-request-template-selector-similar-to-issues/171838) [doesn't have a GUI for it](https://github.community/t/multiple-pull-request-templates/1850/12) right now, but you can use the `?template=` parameter to choose the framework PR template:

https://github.com/publicarray/spksrc/compare/master...issue-template?template=framework.md